src/utils/draggable.ts

```diff
@@ -11,45 +11,53 @@ import 'rxjs/add/operator/skip';
 
 import { DraggableEvent } from './DraggableEvent';
 import { utils } from './utils';
+import { IGridsterDraggableOptions } from '../IGridsterDraggableOptions';
 
 export class Draggable {
     element: Element;
 
     dragStart: Observable<DraggableEvent>;
     dragMove: Observable<DraggableEvent>;
     dragStop: Observable<DraggableEvent>;
-
+    // A simple requestAnimationFrame polyfill
+    private raf: Function;
+    private caf: Function;
     private mousemove: Observable<{} | Event> = Observable.merge(
         Observable.fromEvent(document, 'mousemove'),
         Observable.fromEvent(document, 'touchmove', {passive: true})
     ).share();
-
     private mouseup: Observable<{} | Event> = Observable.merge(
         Observable.fromEvent(document, 'mouseup'),
         Observable.fromEvent(document, 'touchend'),
         Observable.fromEvent(document, 'touchcancel')
     ).share();
-
     private mousedown: Observable<{} | Event>;
-
-    private config = {
-        handlerClass: null
+    private config: IGridsterDraggableOptions = {
+        handlerClass: null,
+        scroll: true,
+        scrollEdge: 36
     };
+    // reference to auto scrolling
+    private autoScrollingInterval;
 
-    constructor(element: Element, config: {handlerClass?: string} = {}) {
+    constructor(element: Element, config: IGridsterDraggableOptions = {}) {
         this.element = element;
         this.mousedown = Observable.merge(
             Observable.fromEvent(element, 'mousedown'),
             Observable.fromEvent(element, 'touchstart')
         ).share();
 
-        this.config.handlerClass = config.handlerClass;
+
+        this.config = (<any>Object).assign({}, this.config, config);
```

Comment:
```
this.config = {...this.config, ...config}
```
Spread operator ?

Reply:
I haven't used to this spread operator yet. It does exactly the same but looks less verbose, so I will use your suggestion. 👍